cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -150,25 +148,17 @@ parameter_types! {
 }
 
 // Configure FRAME pallets to include in runtime.
+#[derive_impl(frame_system::config_preludes::ParaChainDefaultConfig as frame_system::DefaultConfig)]
```

Comment:
I don't remember the discussion on this. Is there no way to remove this "as frame_system::DefaultConfig"?

Reply:
Yeah, this was agreed as an acceptable syntax for now. CC: @kianenigma

Reply:
@franciscoaguirre Basti also raised this in another PR. I have added this to the tracking issue and will try to see if this can be removed.